toolkit/tools/internal/file/file.go

```diff
@@ -282,15 +301,38 @@ func copyWithPermissions(src, dst string, dirmode os.FileMode, changeMode bool,
 		}
 	}
 
-	err = shell.ExecuteLive(squashErrors, "cp", "--preserve=mode", src, dst)
+	return
+}
+
+func CopyFileFromFS(srcFS fs.FS, srcFile, dst string, dirmode os.FileMode, filemode os.FileMode) error {
```

Comment:
Could we reference asset, resource, etc. in the function name here, just to make it easier to figure out what its doing for a new reader?

Reply:
Technically `fs.FS` is a generic file system interface, with implementations for both embedded files and OS files. Though realistically this function will only ever be used for embedded files. So, it is probably fine to change the name.